conf/sample/teardown.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Sample script for creating testable Network Service Mesh CRDs
+#
+# NOTE: This assumes minikube for now
+
+# First, delete the network service
+kubectl delete -f conf/sample/networkservice.yaml
+
+# Delete the endpoints
+kubectl delete -f conf/sample/networkservice-endpoint.yaml
+
+# Delete the channel
+kubectl delete -f conf/sample/networkservice-channel.yaml
+
+# Now delete the daemonset
+kubectl delete -f conf/sample/networkservice-daemonset.yaml
```

Comment:
LGTM
nit: this will not clean up CRD objects in Kubernetes. Its not needed now, but may be later for cleaning up.

Reply:
Ok, I see the point.